[MRG] -- Member View, Group View and filter for members

rosygupta opened the pull request:
This PR looks into these issues: 
#50 add Member to group from /members/
#51 Filtering on /members/

-- Edited Member and Group View
-- Added filters for Members

gedankenstuecke commented on Pipfile:
```diff
@@ -29,8 +24,8 @@ tini = "*"
 whitenoise = "*"
 python-dateutil = "*"
-
+django-filters = "*"
```
This needs to be `django-filter` (without the trailing `s`) because `django-filters` is not actually the package we need (otherwise the loading of `django_filters` in `settings.py` fails. I know, it's confusing.

gedankenstuecke commented on Pipfile.lock:
```diff
@@ -18,57 +18,80 @@
+            "index": "pypi",
+            "version": "==2.0.5"
+        },
+        "django-filters": {
```
After editing the `Pipfile` we need to recompile it to update it here too.

rosygupta commented:
Corrected the package mistake. I'm looking into the migrations issue. There's this "unique constraint" issue with `projectMembers` model that I noticed (which might be related to the migrations issue) but couldn't figure out why. I'll dig deep.

rosygupta commented:
With my latest commit, I have done the following:
1. Fixed Date Filter (will be improved further though)
2. Removed the Number filter from `ID`
3. Fixed the member refresh flow. Earlier `Unique Constraint` error popped up since new member data with ID same as old member data was being added. This was due to some code added in #43.
4. Rendering file data and group data on member page done.
5. Feature to `remove from group` also added . to member page. 

Here's the screen while testing these features (just so that it's easier to review ): https://drive.google.com/file/d/1wduuGeIVknV_ttbwyHYT8LHACmbFo2jS/view

gedankenstuecke commented:
Those changes look great! I ran the thing locally on my end and it works nicely and looks great!

(I think there's no reason to upload recordings, I have it setup on my end so that I can just quickly checkout the latest changes from pull requests)

rosygupta commented:
Some improvements possible are: 
1. Show group name instead of group id in group filter (couldn't find a way around that)
2. show an icon to indicate the collapsing nature of table rows.
3. remove the hidden rows from the table - I think 2 and 3 will be possible by adding a css file separately.

gedankenstuecke commented:
I agree for some of the UI changes and will make issues for those :)
